src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/python/PythonInterpreterListEntryUi.ui.xml

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui">
+	
+	<ui:style>
+	
+	.iconContainer {
+		width: 24px;
+		height: 24px;
+		text-align: center;
+		vertical-align: middle;
+	}
+	
+	.icon {
+	}
+	
+	.version {
+		padding: 4px;
+		min-width: 100px;
+	}
+	
+	.path {
+		padding: 4px;
+	}
+	
+	</ui:style>
+	
+	<g:HorizontalPanel>
+		
+		<g:FlowPanel styleName='{style.iconContainer}'>
+			<g:Image styleName='{style.icon}' ui:field="uiIcon_"></g:Image>
```

Comment:
This creates an unlabeled image (no alt text) so a screen reader will read the underlying filename which is a long guid-like identifier.

One option is to use DecorativeImage instead (which sets empty alt text), but since this icon communicates information, we should have a meaningful alt-text, instead, e.g. "Conda", "VirtualEnv", "Python".